### PR TITLE
upgrade to firebase3

### PR DESCRIPTION
**Pull request by mohit**

### CODE REVIEW

File: web/webpack.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ var ExtractTextPlugin = require("extract-text-webpack-plugin");
 
 module.exports = {
   entry: {
-    "javascripts/main-parent.js": "/Users/pam/Documents/reading-challenge/app/web/main.jsx",
+    "javascripts/main-parent.js": "/home/mohit/code/reading-challenge/app/web/main.jsx",
```
Comment (ipam73) [inline web/webpack.config.js]: note:  should fix at some point, but don't care right now

**Comment by ipam73** [comment]
LGTM

File: web/webpack.config.js (hunk 1 above)
Comment (mohit) [inline web/webpack.config.js]: derp. going to remove this.

Comment (mohit) [inline web/webpack.config.js]: removed.

**Pull request merged**